### PR TITLE
Configure redux on project

### PR DESCRIPTION
**Pull request by claudiocassimiro**

### CODE REVIEW

**Review by Vitor-Hugo-Dev — APPROVED** [review]
Good Job!

File: components/NotFound/index.tsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+      <div className={styles.notFoundContainerImage}>
+        <Image
+          src="/images/sad-face.webp"
+          alt="carrinha triste"
+          layout="fill"
+        />
+      </div>
+      <p className={styles.notFoundText}>{term} ainda não está na sua lista</p>
+    </div>
+  );
```
Comment (claudiocassimiro) [inline components/NotFound/index.tsx:39]: Test this guy

**Review by claudiocassimiro — COMMENTED** [review]
Add one commentary

File: components/NotFound/index.tsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  ) : (
+    <div className={styles.notFoundContainer}>
+      <div className={styles.notFoundContainerImage}>
+        <Image
+          src="/images/sad-face.webp"
+          alt="carrinha triste"
+          layout="fill"
+        />
+      </div>
+      <p className={styles.notFoundText}>{term} ainda não está na sua lista</p>
```
Comment (claudiocassimiro) [inline components/NotFound/index.tsx:37]: Remove this term, and use "Este item" in place

File: components/NotFound/index.tsx [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  const cancelTimeout = setTimeout(() => {
+    setIsLoading(false);
+  }, 3000);
+
+  useEffect(
+    () => () => {
+      clearTimeout(cancelTimeout);
+    },
+    [cancelTimeout],
+  );
```
Comment (claudiocassimiro) [inline components/NotFound/index.tsx:20]: Remove it too

**Review by claudiocassimiro — COMMENTED** [review]
More comments

**Pull request merged**